wpiunits/src/main/java/edu/wpi/first/units/BaseUnits.java

```diff
@@ -23,8 +23,8 @@ private BaseUnits() {
   /** The standard unit of mass, grams. */
   public static final Mass Mass = new Mass(1, "Kilogram", "Kg");
```

Comment:
probably worth a separate PR, but the Javadoc here needs fixing

Reply:
Just did it here, it's small and opening a separate PR is overkill